stargaze/chain.json

```diff
@@ -26,21 +30,21 @@
   },
   "codebase": {
     "git_repo": "https://github.com/public-awesome/stargaze",
-    "recommended_version": "v9.1.0",
+    "recommended_version": "v10.0.1",
     "compatible_versions": [
-      "v9.1.0",
-      "v9.0.0"
+      "v10.0.1",
+      "v10.0.0"
     ],
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/public-awesome/mainnet/main/stargaze-1/genesis.tar.gz"
     },
     "versions": [
       {
-        "name": "v9.0.0",
```

Comment:
Can we instead keep v9 in the version history array, but then add v10 as a new version object below?

Reply:
added v9